From d77c5f4199562947cabc89c63b55faee345fd8fd Mon Sep 17 00:00:00 2001
Message-Id: <d77c5f4199562947cabc89c63b55faee345fd8fd.1571832176.git.liu.denton@gmail.com>
In-Reply-To: <cover.1571832176.git.liu.denton@gmail.com>
References: <cover.1571832176.git.liu.denton@gmail.com>
From: Denton Liu <liu.denton@gmail.com>
Date: Wed, 23 Oct 2019 04:44:42 -0700
Subject: [PATCH 3/5] t4108: use `test_config` instead of `git config`
To: Git Mailing List <git@vger.kernel.org>
Cc: Junio C Hamano <gitster@pobox.com>

Since `git config` leaves the configurations set even after the test
case completes, use `test_config` instead so that the configurations are
reset once the test case finishes.

Signed-off-by: Denton Liu <liu.denton@gmail.com>
---
 t/t4108-apply-threeway.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/t4108-apply-threeway.sh b/t/t4108-apply-threeway.sh
index 49739ce8b4..3615256492 100755
--- a/t/t4108-apply-threeway.sh
+++ b/t/t4108-apply-threeway.sh
@@ -70,7 +70,7 @@ test_expect_success 'apply with --3way' '
 '
 
 test_expect_success 'apply with --3way with rerere enabled' '
-	git config rerere.enabled true &&
+	test_config rerere.enabled true &&
 
 	# Merging side should be similar to applying this patch
 	git diff ...side >P.diff &&
-- 
2.24.0.rc0.197.g0926ab8072

